public/chat_header_button.tsx

```diff
@@ -52,6 +53,7 @@ export const HeaderChatButton = (props: HeaderChatButtonProps) => {
   const core = useCore();
   const flyoutFullScreen = sidecarDockedMode === SIDECAR_DOCKED_MODE.TAKEOVER;
   const flyoutMountPoint = useRef(null);
+  usePatchFixedStyle();
```

Comment:
I am thinking, the patched style should only apply when the chatbot is visible right?

Reply:
Yes.

Reply:
Patchstyle contains the logic that when chatbot is closed or docked takeover. When in these cases, we will update a empty style to remove patch style.

Reply:
I see, so the `sidecarConfig$` will emit a null value once the Chatbot is not visible?

Reply:
Sidecar config will emit a new value with isHidden property is true.Then we will update inserted style with ""

Reply:
I did not find the code that we will update the inserted style when config.hidden is true.